core/src/client/snapshot_notify.rs

```diff
@@ -0,0 +1,81 @@
+use ctypes::BlockHash;
+
+use parking_lot::RwLock;
```

Comment:
Please follow the [coding convention](https://github.com/CodeChain-io/codechain/wiki/Coding-Style#imports-order).

Reply:
Fixed.